### PR TITLE
BXC-3236 - Expand readme with usage and installation instructions

### PR DESCRIPTION
**Pull request by bbpennel**
https://jira.lib.unc.edu/browse/BXC-3236

View updated readme here:
https://github.com/UNC-Libraries/cdm-to-boxc-migration-util/blob/bxc-3236-readme/README.md

### CODE REVIEW

**Review by agoslen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,16 +1,113 @@
...
+### Deploying
+In order to deploy the project to a server or the development VM, see the `deploy_migration_util.rb` command from the `boxc-ansible` project. You can deploy uncommitted changes to the utility by providing the `-p` option. For example, `./deploy_migration_util.rb dev -p /path/to/cdm-to-boxc-migration-util` would build and deploy the current state of the migration util located at the provided path.
+
+# Usage
+
+## Basic Usage on Servers
+```
+cdm2bxc.sh -h
+```
+
```
[the remark this answers is not in the document]
Comment (morganem0) [reply README.md:44]: makes sense.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,113 @@
...
+```
+# Initialize a new migration project (named gilmer_demo, from CDM collection gilmer)
+cdm2bxc.sh init -p gilmer_demo -c gilmer
+cd gilmer
+# Export object records from CDM
+cdm2bxc.sh export
+# Index the exported object records
+cdm2bxc.sh index
+# Map objects to Box-c destiantion
+cdm2bxc.sh destinations generate -dd <dest UUID>
```
[the remark this answers is not in the document]
Comment (morganem0) [reply README.md]: the examples are helpful, thanks for adding this.

File: README.md [hunk 3]
```diff
@@ -59,18 +59,24 @@ cdm2bxc.sh -h
 ```
 # Initialize a new migration project (named gilmer_demo, from CDM collection gilmer)
 cdm2bxc.sh init -p gilmer_demo -c gilmer
+
 cd gilmer
```
Comment (krwong) [inline README.md:63]: I think this should be `cd gilmer_demo`.

Comment (bbpennel) [reply README.md]: Thanks, updated

**Comment by bbpennel** [comment]
I'm going to go ahead and merge this later this afternoon unless there are any further suggestions at this time

**Pull request merged**